docs/howto/custom-file-storage.txt

```diff
@@ -138,4 +144,4 @@ Storages are then accessed by alias from from the
 
     from django.core.files.storage import storages
 
-    example_storage = storages["example"]
+    example_storage = storages["example"]
```

Comment:
Revert unrelated change.

Reply:
I originally added to the end of the file, deleted what I had, but left a trailing empty line by accident.  I then removed the extra line in a later commit.  The code has not been modified in any way.